pokemongo_bot/__init__.py

```diff
@@ -174,8 +174,11 @@ def work_on_cell(self, cell, position, work_on_forts=1):
         if worker.work() == WorkerResult.RUNNING:
             return
 
-        if ((self.config.mode == "all" or
-                self.config.mode == "farm") and work_on_forts):
+
+        number_of_things_gained_by_stop = 5
+
+        if ((self.get_inventory_count('item') < self._player['max_item_storage'] - 5) and
```

Comment:
Yeah, I think I've seen someone say 7 on reddit. I'd be happy to set it to that, but I was kinda trying to go with the average case instead of the best case.